src/main/resources/altoclef.mixins.json

```diff
@@ -2,7 +2,7 @@
   "required": true,
   "minVersion": "0.8",
   "package": "adris.altoclef.mixins",
-  "compatibilityLevel": "JAVA_8",
+  "compatibilityLevel": "JAVA_21",
```

Comment:
This is a breaking change, but is not marked as such anywhere on the PR

Reply:
I have added

[x] This PR is a breaking change (e.g. methods or parameters removed/renamed).

to the PR info so I'll mark this as resolved